heterarchy.litcoffee

```diff
@@ -12,9 +12,9 @@ Dylan paper](http://192.220.96.201/dylan/linearization-oopsla96.html).
     {head, tail, map, find, some, without, isEmpty, every, memoize, reject,
      partial, isEqual, reduce} = require 'underscore'
 
-    assert = (value, error) ->
+    assert = (value, errorMessage) ->
         if not value
-            throw new Error(if error? then error else "Assertion failed")
+            throw new Error(errorMessage)
```

Comment:
Yes, usability-wise it was better (or at least more generic). I changed this, because the (I believe) `error?` was always true. Thus, `test-coverage` said one code branch was never executed.
Maybe the better way is to add an assertion call to the test spec. 😉